### PR TITLE
docs: update debug.md to match the new trace output

### PR DESCRIPTION
**Pull request by jbech-linaro**
This is the updates needed here due to the trace changes in https://github.com/OP-TEE/optee_os/pull/2011.

### CODE REVIEW

File: docs/debug.md [hunk 1]
```diff
@@ -415,107 +415,111 @@ $ echo 0 > options/graph-time
...
-The above dump was triggered by the TA trying to write to a global const
-variable. Such variables are stored in the `.rodata` section of the ELF binary,
-and loaded into a read-only section of memory.
-
-OP-TEE provides a helper script called `symbolize.py` to facilitate the
-analysis of such issues. It is located in the OP-TEE OS source tree, and is
-also copied to the TA development kit. Whenever you are confronted with an error
-message reporting a serious error and showing a "Call stack:" line, you may use
-the symbolize script.
+  read or write permission fault).
```
Comment (etienne-lms) [inline docs/debug.md]: Alignment issues also?

File: docs/debug.md [hunk 2]
```diff
@@ -415,107 +415,111 @@ $ echo 0 > options/graph-time
...
+D/TC:0 tee_ta_close_session:402 tee_ta_close_session(0xe07be98)
+D/TC:0 tee_ta_close_session:421 Destroy session
+D/TC:0 tee_ta_close_session:447 Destroy TA ctx
+```
+
+The above dump was triggered by the TA when entering an irrecoverable error
+ending up in a `TEE_Panic(0)` call.
+
+OP-TEE provides a helper script called `symbolize.py` to facilitate the analysis
+of such issues. It is located in the OP-TEE OS source tree, and is also copied
```
Comment (etienne-lms) [inline docs/debug.md]: remove coma.

s/copied to/installed with/ ?

File: docs/debug.md (hunk 1 above)
Comment (jbech-linaro) [reply docs/debug.md]: Yes, your are probably right.

File: docs/debug.md (hunk 2 above)
Comment (jbech-linaro) [reply docs/debug.md]: It's Jerome's wording and I think copied to is fine here, it's not that we are running something like "make install" (so I'll keep the current wording).

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Pull request merged**